app/templates/views/signin.html

```diff
@@ -28,7 +28,7 @@ <h1 class="font-body-2xl margin-bottom-3">You need to sign in again</h1>
       {% endif %}
     {% else %}
       <h1 class="font-body-2xl margin-bottom-3">Sign in</h1>
-      {% if notify_env_is_dev %}
+      {% if notify_env_is_dev_or_staging %}
```

Comment:
And then here as well?

```suggestion
      {% if login_gov_enabled %}
```

Reply:
I suppose.  It's going to be replaced anyway.